beta-src/src/utils/state/getPhaseKey.ts

```diff
@@ -3,8 +3,11 @@ import GameStatusResponse from "../../state/interfaces/GameStatusResponse";
 import { IContext } from "../../models/Interfaces";
 
 const getPhaseKey = function (
-  data: GameOverviewResponse | GameStatusResponse | IContext,
+  data: GameOverviewResponse | GameStatusResponse | IContext | undefined,
 ): string {
+  if (!data) {
```

Comment:
Err, need to be careful here because "<BAD> === <BAD>", which is exactly what we want to avoid.

I think we should return `None` here and then the consistency check needs to explicitly check if any of the phase keys are `None`.